lib/doc/jsdoc-template/docs.html.handlebars

```diff
@@ -54,7 +54,10 @@
         <section class="card" id="{{name}}">
             <h2>
                 <a tabindex="2" class="name" href="#{{name}}">{{name}}</a>
-                <span class="label label-category pull-right">{{category}}</span>
+                <span class="pull-right">
+                        <span class="label label-category">{{category}}</span>
+                        <a target="_blank" title="View source on GitHub" href="https://github.com/ramda/ramda/tree/v{{../version}}/src/{{name}}.js"><small class="glyphicon glyphicon-new-window"></small></a>
```

Comment:
It's handlebars' way of referencing the parent scope (`docs` in this instance), because the cards are declared inside `{{#each docs}}`.